modules/core/05-port/types/module.go

```diff
@@ -2,8 +2,8 @@ package types
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
-
 	capabilitytypes "github.com/cosmos/ibc-go/modules/capability/types"
```

Comment:
interesting, this doesn't fail the linter? (for unformatted imports?)

Reply:
I expect it probably does! Seems like there's a big queue atm for runners. Was about to push a `make lint-fix` commit which changes these lines